### ISSUE TITLE
XSLT needs revision to address revised encoding of date information in custodial events

### ISSUE THREAD

**Issue by paregorios** [DCLP/dclpxsltbox#43]
At present, the DCLP version of the PN XSLT files constructs a textual representation of each custodial event that [draws upon date information encoded in the `from` and `to` attributes on the `<custEvent>` element](https://github.com/DCLP/navigator/blob/xslt-master/pn-xslt/metadata.xsl#L687). Some `<custEvent>` elements in the data now have no `from` or `to` attributes. Some of these make use instead of the single `when` attribute instead (e.g., ). Issues #41 and #42, which address the encoding of alternative dates and date uncertainty in custodial events, will necessitate upgrades to the current XSLT logic in order to handle all aspects. The encoding philosophy discussion they have prompted (about whether storing all this information only in attributes is the right way to go) may well necessitate a complete refactoring of this portion of the XSLT.

**Comment by paregorios** [comment]
No longer blocked by #41 but waiting for #42.

**Comment by paregorios** [comment]
deferring per dependencies on #41 and #42, which have been deferred.
